pkgs/tools/text/ugrep/default.nix

```
@@ -9,15 +9,15 @@
 , zlib
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "ugrep";
-  version = "3.12.1";
+  version = "3.12.2";
 
   src = fetchFromGitHub {
     owner = "Genivia";
-    repo = pname;
-    rev = "v${version}";
-    hash = "sha256-kpLzv8nHye1XStHABmZHXQ2Gn+g06BFeN3u47bkKbzU=";
+    repo = finalAttrs.pname;
```

overriding pname should not override the repo

```suggestion
    repo = "ugrep";
```

---

Done. Thanks!